src/components/PopupWithImage.js

```diff
@@ -7,15 +7,8 @@ export default class PopupWithImage extends Popup {
     this._previewImageCaption = document.querySelector(".modal__preview-title");
   }
 
-  // open({ name, link }) {
-  //   this._previewImageElement.alt = name;
-  //   this._previewImageCaption.textContent = name;
-  //   this._previewImageElement.src = link;
-  //   super.open();
-  // }
-
   open(cardData) {
-    this._previewImageElement.alt = "Image of ${cardData.name}";
+    this._previewImageElement.alt = 'Image of ${cardData.name}';
```

Comment:
> I'm assuming I just change this to a single quote?

no) you need to use special quotes called  **backtick (`)**

You can copy this code here and paste it and you'll see how it looks in the project

```js

    this._previewImageElement.alt = `Image of ${cardData.name}`;


```




More info here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
updated.